syntax = "proto3";

service EVSales {
  rpc GetSalesFiltered (SalesFilterRequest) returns (SalesReply);
}

message SalesFilterRequest {
  string country = 1;
  int32 year = 2;
  string mode = 3;
}

message SalesReply {
  repeated string sales_xml = 1;
}
